livechat/reports/api/v35.py

```diff
@@ -378,3 +378,34 @@ def performance(self,
         return self.session.post(f'{self.api_url}/agents/performance',
                                  json=payload,
                                  headers=headers)
+
+
+# Tags
+
+    def chat_usage(self,
```

Comment:
Wondering whether section is enough or should be named `tags_chat_usage` 🤔

Reply:
`tags_chat_usage` would be more meaningful, we don't get enough context when importing this method when using the package